### PR TITLE
chore: Turned on QR scanner

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: electrum/gui/qt/main_window.py [hunk 1]
```diff
@@ -766,7 +766,7 @@ def add_toggle_action(view_menu, tab):
         raw_transaction_menu.addAction(_("&From text"), self.do_process_from_text)
         raw_transaction_menu.addAction(_("&From the blockchain"), self.do_process_from_txid)
         # fixme reading transaction from QR doesn't work
```
Comment (cloudjrojek) [inline electrum/gui/qt/main_window.py:768]: if it works, remove the comment please

File: electrum/gui/qt/paytoedit.py [hunk 2]
```diff
@@ -58,12 +58,12 @@ class PayToLineError(NamedTuple):
     is_multiline: bool = False
 
         #todo uncomment when QR Read will be work fine
```
Comment (cloudjrojek) [inline electrum/gui/qt/paytoedit.py:60]: if its working, remove the comment please

File: electrum/gui/qt/qrtextedit.py [hunk 3]
```diff
@@ -14,10 +14,10 @@ def __init__(self, text=None, *, config: SimpleConfig):
...
         self.setReadOnly(True)
 #todo uncomment when QR Read gonna be fixed
-#        icon = "qrcode_white.png" if ColorScheme.dark_scheme else "qrcode.png"
-#        self.addButton(icon, self.qr_show, _("Show as QR code"))
-#
-#        run_hook('show_text_edit', self)
+        icon = "qrcode_white.png" if ColorScheme.dark_scheme else "qrcode.png"
+        self.addButton(icon, self.qr_show, _("Show as QR code"))
+
+        run_hook('show_text_edit', self)
```
Comment (cloudjrojek) [inline electrum/gui/qt/qrtextedit.py:20]: if its working, remove the comment please

File: electrum/gui/qt/qrtextedit.py [hunk 4]
```diff
@@ -45,10 +45,10 @@ def __init__(self, text="", allow_multi=False, *, config: SimpleConfig):
         self.config = config
         self.setReadOnly(False)
         #todo uncomment when ReadFile handle only text extension and QR Read gonna be fixed
```
Comment (cloudjrojek) [inline electrum/gui/qt/qrtextedit.py:47]: remove the part about missing QR Read

**Review by cloudjrojek — CHANGES_REQUESTED** [review]

File: electrum/gui/qt/main_window.py (hunk 1 above)
Comment (kj-crypto) [reply electrum/gui/qt/main_window.py]: done

File: electrum/gui/qt/paytoedit.py (hunk 2 above)
Comment (kj-crypto) [reply electrum/gui/qt/paytoedit.py]: done

File: electrum/gui/qt/qrtextedit.py (hunk 3 above)
Comment (kj-crypto) [reply electrum/gui/qt/qrtextedit.py]: done

File: electrum/gui/qt/qrtextedit.py (hunk 4 above)
Comment (kj-crypto) [reply electrum/gui/qt/qrtextedit.py]: done

**Review by cloudjrojek — APPROVED** [review]

**Pull request merged**